indexing-service/src/main/java/io/druid/indexing/common/task/VersionConverterTask.java

```diff
@@ -84,17 +85,19 @@ private static VersionConverterTask createFromJson(
       @JsonProperty("dataSource") String dataSource,
       @JsonProperty("interval") Interval interval,
       @JsonProperty("segment") DataSegment segment,
-      @JsonProperty("indexSpec") IndexSpec indexSpec
+      @JsonProperty("indexSpec") IndexSpec indexSpec,
+      @JsonProperty("force") Boolean force
```

Comment:
given that we haven't documented this task, it might be nice to add some JavaDocs while we're at it